blueprint/api.apib

```diff
@@ -2174,6 +2315,30 @@ This endpoint allows you to check whether a username is valid (by running a vali
 + data(Slugged Route Organization Resource)
 + include JSON API Version
 
+## Stripe Plan Attributes (object)
++ amount: 999 (number)
++ created: `2016-07-08T03:03:51.967Z` (string)
++ name: `Emerald Personal` (string)
```

Comment:
I also think you should be writing descriptions for all the attributes, like `amount: 999 (number) - Describes the amount of a plan in cents.`

Also, for the `name` here, the example doesn't really resonate with our application. We should come up with a naming convention for plans here. I think `Monthly Donation` is sufficient.